docs/react-migration-guide.md

```diff
@@ -1,5 +1,207 @@
 # React hooks upgrade / migration guide
 
+## Version ?? to ??
```

Comment:
Previous iterations of this migration guide used versions used by separate react-hooks library, which was not part of ably-js repo (see line https://github.com/ably/ably-js/pull/1698/files#diff-f8ce6205300ecf411be3e91af62c0ecbaf88971e7cd082fc9193ef20cdea32eeR205), so previous guide was for `Version 2.x to 3.x`.

What do we want write now here?
React Hooks are now part of ably-js and follow its versioning.

Reply:
the "2.x to 3.x" thing shouldn't have been there in the first place (probably my bad) - we were originally making some breaking changes as part of an `@ably-labs/react-hooks` v3 major release but instead decided to merge the hooks into the core `ably` package so there never was a 3.x release in the end.

I think we should change the header of the previous migration guide to "`@ably-labs/react-hooks` to 1.x" and then the new migration guide can simply be labelled "1.x to 2.x"

Reply:
how about "`@ably-labs/react-hooks` to `ably-js 1.x`"? so it's a bit more descriptive that we moved to `ably-js` repo

Reply:
sounds good to me, but i would go for `ably` rather than `ably-js` since that's the name of the npm package

Reply:
yea, makes sence! will do `ably`

Reply:
changed in https://github.com/ably/ably-js/commit/1c5d80da35f5ca560afb988b4df7cfd7ce23d6ef